bugnote_view_inc.php

```diff
@@ -216,20 +220,36 @@
 
 				# show edit button if the user is allowed to edit this bugnote
 				if( $t_can_edit_bugnote ) {
-					print_button( 'bugnote_edit_page.php?bugnote_id='.$t_bugnote->id, lang_get( 'bugnote_edit_link' ) );
+					print_button(
+						'bugnote_edit_page.php',
+						lang_get( 'bugnote_edit_link' ),
+						array( 'bugnote_id' => $t_bugnote->id ),
+						OFF );
```

Comment:
Why is this OFF?  If that is the correct behavior, then add a comment to explain.